client/webserver/site/src/js/state.ts

```diff
@@ -7,6 +7,8 @@ const pwKeyCK = 'sessionkey'
 // utilities for setting and retrieving cookies and storing user configuration
 // to localStorage.
 export default class State {
+  static orderDisclaimerAckedLK = 'ordAck'
```

Comment:
I'm sure you see @norwnd, but please note this for https://github.com/decred/dcrdex/pull/1975

Reply:
yep (and appreciate you pointing this out! better safe...), 

I feel like those commits in https://github.com/decred/dcrdex/pull/1975/commits need to be squashed before I rebase (and we merge into master),

but I believe I should get final approval in https://github.com/decred/dcrdex/pull/1975 first, so that reviewers can focus on simple diffs for latests commits, just to clarify I've got your review-process approach right @chappjc ?